src/Components/Modal.js

```diff
@@ -0,0 +1,198 @@
+import styled from 'styled-components';
+// import img from "./modal.jpg"
+// import { mdiWikipedia } from '@mdi/js';
+import {Icon} from '@iconify/react';
+
+export default function Modal({open, onClose, content}) {
+  console.log(content);
+  return (
+    open && (
+      <>
+        <Styled_Close_Button type="button" onClick={onClose}>
+          <Icon
+            icon="ant-design:close-square-outlined"
+            width="45"
+            height="45"
+          />
+        </Styled_Close_Button>
```

Comment:
Please check your code quality issue. Your component names should be written in PascalCase (that's almost the same as camelCase, just the first letter needs to be capital).

```suggestion
        <StyledCloseButton type="button" onClick={onClose}>
          <Icon
            icon="ant-design:close-square-outlined"
            width="45"
            height="45"
          />
        </StyledCloseButton>
```